tests/test_completion/test_completion.py

```diff
@@ -3,9 +3,14 @@
 import sys
 from pathlib import Path
 
+import pytest
+
 from docs_src.commands.index import tutorial001 as mod
 
 
+@pytest.mark.skipif(
+    not sys.platform.startswith("linux"), reason="Test designed for Linux/bash"
+)
```

Comment:
Maybe we could move this and the one below to `tests/utils.py`, there's a `@needs_py310` there, we could have a `@needs_linux` with this that we can re-use in both places.

Reply:
Yep! That's perfect! :rocket: